README.md

```diff
@@ -7,26 +7,54 @@ This repository accompanies Pelm's [**quickstart guide**](https://pelm.readme.io
 Clone the repository
 
 ```bash
-git clone https://github.com/Pelm-Technologies/quickstart.git
-cd quickstart
-```
+  git clone https://github.com/Pelm-Technologies/quickstart.git
```

Comment:
Can we create separate README's for `react` and `vanilla_js`. That'd be easier to follow.

And we can tell them to `cd` into the appropriate folder and go from there.

Reply:
Added!